GoogleApiWrappers/src/Google.Apis.Storage.v1.ClientWrapper/StorageClient.DownloadObject.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright 2015 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Download;
+using Google.Common;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Object = Google.Apis.Storage.v1.Data.Object;
+
+namespace Google.Apis.Storage.v1.ClientWrapper
+{
+    public partial class StorageClient
+    {
+        // Note: documentation will be provided when we've worked out if we really want 16 methods here.
+        // See issue #21.
+
+        public void DownloadObject(string bucket, string sourceName, Stream destination)
```

Comment:
"path" would suggest a local file system path, to me at least.
"objectName" would be okay in this case I suppose - I just wanted to make the "source" and "destination" really clear. (In upload I'd have destinationName, in the current scheme.)

Very happy to debate this though - parameter names are part of the API, after all.